CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 0.33.0
```

Comment:
There are many other changes in this version.

Reply:
You are correct. None of the other commits made an entry to `CHANGELOG` and I forgot to check how many commits have been made since 0.32.0. I'll have a look and add them here.

Reply:
Yeah, usually I update the CHANGELOG before making a release. 

I can do it tomorrow, unless you want to take a stab at it.

Reply:
@casperisfine thanks, that would be great! you are intimately more familiar with the codebase, and were there to review all the changes as well. It will take you a lot less time, than it would take me.